docs/cli/server.md

```diff
@@ -116,10 +118,6 @@ The ```bor server``` command runs the Bor client.
 
 - ```cache.trie```: Percentage of cache memory allowance to use for trie caching (default: 15)
 
-- ```cache.trie.journal```: Disk journal directory for trie cache to survive node restarts (default: triecache)
```

Comment:
Any reason for these removals?

Reply:
+1 we should keep this. Try using `make docs` to auto-generate server.md

Reply:
Hey @marcello33 , @cffls - this was removed by the `make docs` command only.

And this flag was removed during the [geth upstream merge](https://github.com/maticnetwork/bor/commit/6aed36d9d1ed1bc3f98605d6858c126c8429b1fc), because geth removed it.

The reason I skipped `make docs` during that time is because [Leo's recent changes](https://github.com/maticnetwork/bor/pull/1063) were not present in that branch, which improved `make docs` command.